fdw/pxf_header.c

```diff
@@ -180,29 +181,29 @@ AddTupleDescriptionToHttpHeader(CHURL_HEADERS headers, Relation rel)
 	/* Get tuple description itself */
 	tuple = RelationGetDescr(rel);
 
-	/* Convert the number of attributes to a string */
-	pg_ltoa(tuple->natts, long_number);
-	churl_headers_append(headers, "X-GP-ATTRS", long_number);
-
 	/* Iterate attributes */
-	for (int i = 0; i < tuple->natts; ++i)
+	for (i = 0, attrIx = 0; i < tuple->natts; ++i)
 	{
 		Form_pg_attribute attr = TupleDescAttr(tuple, i);
 
+		// Ignore dropped attributes
+        if(attr->attisdropped)
```

Comment:
```suggestion
        if (attr->attisdropped)
```
also check alignment